packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -315,7 +278,7 @@ export default {
       this.sharedParentDir = sharePathParentOrCurrent
     }
   },
-  mounted() {
+  async mounted() {
```

Comment:
`async` doesn't make sense if you don't `await` anything. Probably a dev leftover?

Reply:
sorry, missed the await, both are async